doc/en/user/source/tutorials/GetFeatureInfo/html.rst

```diff
@@ -0,0 +1,161 @@
+HTML output format
+===================
+
+The default HTML output is a sequence of titled tables, each one for a different layer. The following example shows the default output for the tiger-ny basemap (included in the above cited releases, and onwards).
+
+.. figure:: default.png
+   :align: center
+
+   *Default Output*
+
+Standard Templates
```

Comment:
Wondering how much of this new content overlaps with the existing feature info Freemarker tutorial?

https://geoserver.geo-solutions.it/downloads/releases/2.8.x-ld/doc/tutorials/GetFeatureInfo/index.html

Reply:
This is actually that one. About getfeatureinfo there is only a section in the WMS reference page https://docs.geoserver.org/latest/en/user/services/wms/reference.html, and the that tutorial.
So I added the geojson doc page along that one in the tutorials section. In the last commit I've added also two example request and two example response in the geojson page.